src/components/Transaction/index.tsx

```diff
@@ -34,15 +42,67 @@ const TransactionCell = ({ cell, address }: { cell: any; address?: string }) =>
   )
 }
 
+const CellContainerComponent = ({
+  cells,
+  address,
+  pageSize,
+  children,
+}: {
+  cells: [any]
+  address?: string
+  pageSize: number
+  children?: ReactNode
+}) => {
+  const [count, setCount] = useState(pageSize)
+  const onClickLoadMore = () => {
+    setCount(Math.min(cells.length, count + pageSize))
+  }
+  return (
+    <CellContainer>
+      {cells &&
+        cells.map((cell: InputOutput, idx: number) => {
+          return idx < count && cell && <TransactionCell cell={cell} address={address} key={cell.id} />
+        })}
+      {count < cells.length && (
+        <button type="button" onClick={onClickLoadMore}>
+          Load More
+        </button>
+      )}
+      {children}
+    </CellContainer>
+  )
+}
+
+const getCapacityChange = (transaction: any, address?: string) => {
+  if (!transaction) return 0
+  let capacity: number = 0
+  for (let index = 0; index < transaction.display_inputs.length; index++) {
```

Comment:
Maybe `for..in` is OK. For this loop, index is no use.

Reply:
https://github.com/nervosnetwork/ckb-explorer-frontend/pull/38/commits/a4b213041239e957352fa328ed81cec2c78ace30